app/src/main/java/fr/free/nrw/commons/Media.java

```diff
@@ -58,17 +65,40 @@ public Media createFromParcel(Parcel parcel) {
     protected String licenseUrl;
     protected String creator;
     protected ArrayList<String> categories; // as loaded at runtime?
+    /**
+     * Depicts is a feature part of Structured data. Multiple Depictions can be added for an image just like categories.
+     * However unlike categories depictions is multi-lingual
+     * Some expected properties include
```

Comment:
Please remove these 3 lines, as we moved this out of the scope of this GSoC. Thanks!